web/frontend/src/components/slam.vue

```diff
@@ -141,23 +214,6 @@ const scheduleRefresh2d = (name: string, time: string) => {
   slam2dTimeoutId = window.setTimeout(timeoutCallback, Number.parseFloat(time) * 1000);
 };
 
-const scheduleRefresh3d = (name: string, time: string) => {
```

Comment:
Are we removing 3d PCD features as part of this PR?

Reply:
i will follow up with peter b about this.